aws/session/shared_config.go

```diff
@@ -308,6 +338,30 @@ func (cfg *sharedConfig) setFromIniFiles(profiles map[string]struct{}, profile s
 		cfg.SourceProfile = srcCfg
 	}
 
+	// If the profile contains an SSO session parameter, the session MUST exist
+	// as a section in the config file. Load the SSO session using the name
+	// provided. If the session section is not found or incomplete an error
+	// will be returned.
+	if cfg.hasSSOTokenProviderConfiguration() {
+		skippedFiles = 0
```

Comment:
does V1 only have a concept of multiple shared config files?

Reply:
Yep, in v2's [sharedConfig loading](https://github.com/aws/aws-sdk-go-v2/blob/8fa4abb61a3c25c2b2eb99eb45bb697bb987957a/config/shared_config.go#L496), sections will be extracted from config files first, but in v1 files will be used directly to load shared config